readme.md

```diff
@@ -47,6 +47,21 @@ Java files will be generated under `where/to/generate/java/files/src/main/java/s
 
 To debug, add `--java.debugger` to the argument list. The JVM will suspend at the beginning of the execution. Then attach a remote debugger in your IDE to `localhost:5005`. **Make sure you detach the debugger before killing the AutoRest process. Otherwise it will fail to shutdown the JVM and leave it orphaned. (which can be killed in the Task Manager)**
 
+# Recommended Settings
+
+- `client-logger: true`, Exception is required to be thrown via `ClientLogger`.
```

Comment:
The name of the setting doesn't reflect what it does. Can we instead call the setting as `throw-using-client-logger`? Given that this is a relatively new flag, I don't think any new services would have used it yet and can be updated.

Reply:
I've remove it for now. If dev uses `client-side-validations`, CI would tell them what's wrong.